array/models/primitives/TypeContactNumber.model.bxb

```diff
@@ -0,0 +1,3 @@
+integer (TypeContactNumber) {
+  description (must be a TEN DIGIT number and not starting at zero, xxx-xxx-xxxx)
```

Comment:
The description doesn't really help if there is really no validation logic in the code.

Reply:
Agreed. Do you do validation in a view somewhere? (For example: https://bixbydevelopers.com/dev/docs/reference/type/input-view.render.form.elements.text-input.pattern.regex) Or maybe not use an integer type. The description is also confusing because your description says `xxx-xxx-xxxx`, but the dashes (`-`) aren't actually allowed.

Reply:
Agree and removed.